docs/workflow/ci/disabling-tests.md

```diff
@@ -0,0 +1,71 @@
+# Disabling tests
+
+This document describes how to disable a test from running in the continuous integration (CI) test system.
+
+## Why disable tests?
+
+Tests are either disabled permanently or temporarily:
+- Permanently: the test is never expected to run in a certain configuration, due to the design of the test or product.
+- Temporarily: the test is failing, and needs to be disabled so a test run isn't continuously made "noisy" by the
+existence of a failing test in the results. These tests are expected to be re-enabled when a bug is fixed or
+feature is implemented.
+
+## Runtime or libraries?
+
+There are two main sets of tests in the repo: runtime tests in the src/tests tree, and libraries tests which
```

Comment:
are there any CLR specific or Mono specific test trees now, or are they all just subsets of the runtime tests in src/tests?

Reply:
IIUC, it is the same test tree, but I don't know if Mono has something in addition.